libraries/SITL/SIM_Aircraft.cpp

```diff
@@ -296,30 +297,23 @@ double Aircraft::rand_normal(double mean, double stddev)
 {
     static double n2 = 0.0;
     static int n2_cached = 0;
-    if (!n2_cached)
-    {
+    if (!n2_cached) {
         double x, y, r;
         do
         {
-            x = 2.0*rand()/RAND_MAX - 1;
-            y = 2.0*rand()/RAND_MAX - 1;
-
+            x = 2.0 * rand()/RAND_MAX - 1;
+            y = 2.0 * rand()/RAND_MAX - 1;
             r = x*x + y*y;
-        }
-        while (is_zero(r) || r > 1.0);
-        {
```

Comment:
I think so. I'm very surprised it was written like that in the first place. You changed it to implement the same code but formatted correctly, which is much better. The question is was the real code what we actually we wanted? If noyt, that's a different PR.